games/openttd/patch-build.diff

```diff
@@ -0,0 +1,36 @@
+diff --git config.lib config.lib
+index 304b86e..03c1536 100644
+--- config.lib
++++ config.lib
+@@ -667,7 +667,7 @@ check_params() {
+ 
+ 	if [ "$enable_lto" != "0" ]; then
+ 		# GCC 4.5 outputs '%{flto}', GCC 4.6 outputs '%{flto*}'
+-		has_lto=`($cxx_build -dumpspecs 2>&1 | grep '\%{flto') || ($cxx_build -help ipo 2>&1 | grep '\-ipo')`
++		has_lto=`($cxx_build -dumpspecs 2>&1 | grep '\%{flto') || ($cxx_build -help ipo 2>&1 | grep '\-ipo') || ($cxx_build --help 2>&1 | grep 'lto.*thin')`
```

Comment:
This seems to be an upstream-able change. Did you submit it?

Reply:
Yeah, it's here: https://github.com/OpenTTD/OpenTTD/pull/8182